README.md

```diff
@@ -53,20 +53,33 @@ and where.
 ### Manual Installation
 
 1) Pull down the code from https://github.com/SomeOddCodeGuy/OfflineWikipediaTextApi
+   `git clone https://github.com/SomeOddCodeGuy/OfflineWikipediaTextApi`
 2) Open command prompt and navigate to the folder containing the code
+   `cd OfflineWikipediaTextApi`
 3) Optional: create a python virtual environment.
    1) Windows: `python -m venv venv`
    2) MacOS/Linux: `python3 -m venv venv`
 4) Optional: activate python virtual environment.
    1) Windows: `venv\Scripts\activate`
    2) MacOS/Linux: `venv/bin/activate`
+   3) Fish shell: `venv/bin/activate.fish`
 5) Pip install the requirements from requirements.txt
    1) Windows: `python -m pip install -r requirements.txt`
-   2) Linux/MacOS: `python3 -m pip install -r requirements.txt`
+   2) Linux/MacOS: `python -m pip install -r requirements.txt`
```

Comment:
Would you mind breaking out MacOS / Linux as separate line items if python3 is an issue for Linux? 

In the past I've had issues with just `python` working with my MacOS as it didn't point to the appropriate Python 3 interpreter, but I did confirm that python3 is working appropriately for Mac within the venv for this; I've got the API running via the run_macos script at this moment. 

Splitting mac and linux out may make more sense in that case.